templates/service_catalog/common/instance-details.html

```diff
@@ -102,6 +103,9 @@ <h3 class="card-title">Details</h3>
                             <div class="tab-pane" id="spec">
                                 <pre><code class="language-json">{{ instance.spec | pretty_json }}</code></pre>
                             </div>
+                            <div class="tab-pane" id="user-spec">
```

Comment:
if needed to hide in HTML generated by user

Reply:
done